e2e/test/scenarios/embedding/embedding-reproductions.cy.spec.js

```diff
@@ -494,6 +504,109 @@ describe("issues 20845, 25031", () => {
   });
 });
 
+describe.skip("27643", { tags: "@external" }, () => {
+  const PG_DB_ID = 2;
```

Comment:
I see we hard-coded this in a lot of places. We actually have https://github.com/metabase/metabase/blob/9df1e4a92d38db1d5aaa77aed5a2e176bcb83243/e2e/support/cypress_data.js#L192

The name suggests it's only for writeable tests, so I guess that's why a lot of other tests doesn't use this variable since it could be misleading.